src/main/java/net/sf/jabref/gui/keyboard/KeyBinding.java

```diff
@@ -25,7 +25,7 @@
     DECREASE_TABLE_FONT_SIZE("Decrease table font size", Localization.lang("Decrease table font size"), "ctrl MINUS"),
     DELETE_ENTRY("Delete entry", Localization.lang("Delete entry"), "DELETE"),
     EDIT_ENTRY("Edit entry", Localization.lang("Edit entry"), "ctrl E"),
-    EDIT_PREAMBLE("Edit preamble", Localization.lang("Edit preamble"), "ctrl P"),
+    EDIT_PREAMBLE("Edit preamble", Localization.lang("Edit preamble"), "alt S"),
```

Comment:
I have to agree with you, I'll revert that change. To be honost I don't remember why that was changed.